src/utils/config_manager.py

```
@@ -85,7 +85,7 @@ def _validate_and_migrate(self, raw: Dict[str, Any]) -> Dict[str, Any]:
         try:
             return AppConfig(**migrated).dict()
         except ValidationError as e:
-            raise ConfigError("Config validation failed", context={"errors": e.errors()})
+            raise ConfigError("Config validation failed", details={"errors": e.errors()})
```

Ruff (0.14.11)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)